src/components/common/modal/ModalActionsByFeedType.js

```diff
@@ -40,15 +40,24 @@ const ModalActionsByFeedType = ({ theme, styles, item, handleModalClose, navigat
           .cancelOTLByTransactionHash(item.id)
           .catch(e => {
             userStorage.recoverEvent(item.id)
-            log.error('cancel payment failed', e.message, e)
+            log.error('cancel payment failed', {
```

Comment:
wrong logging convention, fix in all places

Reply:
@sirpy What is the correct convention? ('human message', e.message, e) like that?

Reply:
yes

Reply:
Done